airbyte-webapp/src/packages/cloud/lib/domain/users/UserService.ts

```diff
@@ -46,6 +50,13 @@ export class UserService extends AirbyteRequestService {
     });
   }
 
+  public async resendWithSignInLink({ email }: { email: string }): Promise<void> {
+    this.fetch(`v1/web_backend/cloud_workspaces/resend_with_signin_link`, {
+      email,
+      continueUrl: window.location.href,
```

Comment:
I think it would be helpful to put a comment here, that the `continueUrl` is at the moment not used (as far as I could tell) in the the Frontend, and we always bring the user after registration to their default home page.